### ISSUE TITLE
Contributor console modal when company found but hasn't signed CCLA

### ISSUE THREAD

**Issue by pranab-bajpai** [communitybridge/easycla#562]
## Summary

Describe the bug - a clear and concise description of what the bug is.
![image](https://user-images.githubusercontent.com/57362064/73995372-016d2c80-491e-11ea-9675-4c209142f226.png)

This modal needs two changes
1. Where it says "Company Administrator", it should say "CLAManager"
2. It should show the name of the project for which company hasn't signed CCLA

## Expected behavior

This modal needs two changes
1. Where it says "Company Administrator", it should say "CLAManager"
2. It should show the name of the project for which company hasn't signed CCLA

## Screenshots

in summary above

## References

\(Optional\) Provide any code or specification references that would be helpful for the developer implementing this feature.

**Comment by nirupamav** [comment]
Verified and modal is updated
![image](https://user-images.githubusercontent.com/60865903/76077252-61f79400-5fc6-11ea-8552-683e27f6764d.png)
